libraries/chain/include/graphene/chain/protocol/son_wallet.hpp

```diff
@@ -27,6 +27,7 @@ namespace graphene { namespace chain {
         son_wallet_id_type son_wallet_id;
         graphene::peerplays_sidechain::sidechain_type sidechain;
         string address;
+        graphene::peerplays_sidechain::bytes redeem_script;
```

Comment:
Redeem sctipt is already stored in address field, as address and a script are tightly paired. This field is not needed and breaks abstraction, by adding bitcoin specific field.

Reply:
What do you mean exactly? Address contains only redeem script hash.

Reply:
I see now,  the previous version contains json with all info,  I'll fix it,  thank you